planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -990,13 +996,17 @@ SlowDownSection ObstacleStopPlannerNode::createSlowDownSection(
   const double dist_remain, const double dist_baselink_to_obstacle,
   const VehicleInfo & vehicle_info, const double current_acc)
 {
+  mutex_.lock();
   if (!current_velocity_ptr_) {
     // TODO(Satoshi Ota)
+    mutex_.unlock();
     return SlowDownSection{};
   }
 
+  const auto & current_vel = current_velocity_ptr_->twist.twist.linear.x;
```

Comment:
It maybe changed `current_velocity_ptr_->twist.twist.linear.x` after unlock
```suggestion
  const auto current_vel = current_velocity_ptr_->twist.twist.linear.x;
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1330/commits/fc819f857be1b2e812d5a9b062e9e8ba1c7b9bd4